src/styles/main-blocks/_gallery.scss

```diff
@@ -32,6 +33,15 @@
     border: 2px solid $c-own;
     width: 188px;
     height: 56px;
+    transition: color 1s,
```

Comment:
Just change it to `transition: all 1s;`

Reply:
And as far as I remember, transitions should have the same duration throughout the whole page, so it would be wise to create a variable like $transition-duration and use it for every transition.

Reply:
Так, там радилось використовувати однакову тривалість, але мені не подобається коли ховер посилання має період 1s (занадто довго), а ховер кнопки 0,5 (надто швидко). 
Тому в різних місцях, використав різні значення.
Однак, окей, дві змінни, для різних випадків, я згоден, є сенс використати.

Reply:
✓